pkgs/tools/X11/xssstate/default.nix

```diff
@@ -4,29 +4,33 @@
 , libX11
 , libXScrnSaver
 }:
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "xssstate";
   #
   # Use the date of the last commit, since there were bug fixes after the 1.1
   # release.
   #
-  version = "unstable-2022-09-24";
+  version = "1.1-unstable-2022-09-24";
   src = fetchgit {
     url = "https://git.suckless.org/xssstate/";
     rev = "5d8e9b49ce2970f786f1e5aa12bbaae83900453f";
     hash = "sha256-Aor12tU1I/qNZCdBhZcvNK1FWFh0HYK8CEI29X5yoeA=";
   };
 
-  makeFlags = [ "VERSION=${version}" ];
+  installFlags = [
+    "PREFIX=${placeholder "out"}"
+    "VERSION=${finalAttrs.version}"
+  ];
 
-  installFlags = [ "PREFIX=$(out)" ];
+  buildInputs = [
+    libX11
+    libXScrnSaver
+  ];
 
-  buildInputs = [ libX11 libXScrnSaver ];
-
-  meta = with lib; {
+  meta = {
```

Comment:
This is not worth changing

Reply:
Is there a standard for the `meta` attribute? I ask this because on other packages in which I used `meta = with lib; { #...` I was asked by reviewers to drop it in favor of the change that I made here